CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@
 The SPDX specification is maintained by the SPDX [legal][spdx-legal] and [tech][spdx-tech] teams.
 Design and planning is primarily done via the team [mailing][spdx-legal-list] [lists][spdx-tech-list] and meetings.
 
+## Contribution License Agreement
+
+Contributions to this repo are made pursuant to the [SPDX Community Specification Contributor License Agreement 1.0][cla]. You do not need to submit a signed copy of the contributor license agreement; by making a contribution to this repo, you agree to the terms set forth in that agreement.
+
```

Comment:
Propose under the CLA section we add an example how contributor can sign off their commits similar to https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin

Reply:
I'm fine with adding language describing how to submit a DCO sign-off, but I don't think it should be under this section. As noted above, the DCO sign-off isn't needed in order for a contribution to be submitted.

Reply:
Personally, I would also be in favour of including a point about using `Signed-off-by` - it's surely better to have explicit consent than having it implicitly from the license. However, I don't think it necessarily needs to be in this PR :smiley:

Reply:
agree with @swinslow here - we can add that info somewhere, but we should not reply that the inclusion of "Signed-off-by" or lack thereof makes any difference as to the license under which contributions are made. 

as to the wording, personally, I'd lead with the "By making by  a contribution to this repo. . ." and move the "you don't have to sign it" to the end as that is more clarifying/explanatory

Reply:
Sign off is not required by Community-Spec-1.0  but from my understanding of the [SPDX Community Specification CLA](https://github.com/spdx/governance/blob/main/0._SPDX_Contributor_License_Agreement.md) it is - " including my sign-off".  

The section "In addition, for source code contributions, I certify that: ... these agreements on behalf of that employer." is slightly modified version of https://developercertificate.org/ so from a community perspective I expect to have to sign off my contributions. 

We always signed off contribution for spdx-spec repository and even have DCO check running for all pull requests.